internal/plugins/ansible/v1/api.go

```diff
@@ -98,6 +100,10 @@ func (p *createAPISubcommand) UpdateMetadata(cliMeta plugin.CLIMetadata, subcmdM
 func (p *createAPISubcommand) BindFlags(fs *pflag.FlagSet) {
 	fs.SortFlags = false
 	fs.StringVar(&p.options.CRDVersion, crdVersionFlag, defaultCrdVersion, "crd version to generate")
+	// (not required raise an error in this case)
+	// nolint:errcheck,gosec
+	fs.MarkDeprecated(crdVersionFlag, util.WarnMessageRemovalV1beta1)
```

Comment:
Just one quick question, do we need to mark it as hidden or no need.

Reply:
I think deprecated is enough